<a href="https://colab.research.google.com/github/aleja200105/Coursera_Capstone-M.-Ramirez/blob/main/Agrupaci%C3%B3n_y_Segmentaci%C3%B3n_de_Vecindarios_en_la_Ciudad_de_Toronto%2C_Canad%C3%A1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Vecindarios de Toronto

Se usa un marco de datos antiguo de wikipedia adecuado para su interpretación

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

df_toronto = pd.read_html('https://en.wikipedia.org/w/index.php?title=List_of_postal_codes_of_Canada:_M&oldid=1008658788') # enlace a antigua versión de datos
df_toronto = df_toronto[0]
df_toronto.head(5)

Postal Code           Borough              Neighbourhood
0         M1A      Not assigned               Not assigned
1         M2A      Not assigned               Not assigned
2         M3A        North York                  Parkwoods
3         M4A        North York           Victoria Village
4         M5A  Downtown Toronto  Regent Park, Harbourfront

In [2]:
df_toronto.shape

(180, 3)

In [3]:
# eliminar Not assigned
df_toronto = df_toronto[df_toronto['Borough']!= 'Not assigned']
df_toronto = df_toronto.reset_index(drop=True)
df_toronto.head()

Postal Code           Borough                                Neighbourhood
0         M3A        North York                                    Parkwoods
1         M4A        North York                             Victoria Village
2         M5A  Downtown Toronto                    Regent Park, Harbourfront
3         M6A        North York             Lawrence Manor, Lawrence Heights
4         M7A  Downtown Toronto  Queen's Park, Ontario Provincial Government

In [4]:
df_toronto = df_toronto[df_toronto['Neighbourhood']!= 'Not assigned']
df_toronto.head(5)

Postal Code           Borough                                Neighbourhood
0         M3A        North York                                    Parkwoods
1         M4A        North York                             Victoria Village
2         M5A  Downtown Toronto                    Regent Park, Harbourfront
3         M6A        North York             Lawrence Manor, Lawrence Heights
4         M7A  Downtown Toronto  Queen's Park, Ontario Provincial Government

In [5]:
df_toronto.shape

(103, 3)

Tamaño final del Dataframe es de 103 filas y 3 columnas